mrt/static/js/printouts.js

```diff
@@ -22,12 +22,10 @@ $(function () {
         job_row.addClass('success');
         job_row.find('.status').text(STATUS.FINISHED);
         
-        var filePath = "[?:[a-zA-Z0-9-_\.]+(?:.xls|.xlsx)";
-        
-        var html = 'Success';
-        if (result.toString().match(filePath)) {
+        html = result;
+        if (result.split('/').pop().indexOf('.') > -1)
```

Comment:
I don't think checking for `.` is necessary here, unless I'm missing something? File extensions are not mandatory, and the filename doesn't need to be in the URL for a file download. 

Checking for an absolute URL (starts with `https://` or `http://`) or relative url (starts with with `/`) would be better.